gr-pdu/lib/pdu_split_impl.h

```diff
@@ -0,0 +1,40 @@
+/* -*- c++ -*- */
+/*
+ * Copyright 2021 NTESS LLC.
+ *
+ * This file is part of GNU Radio
+ *
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ *
+ */
+
+#ifndef INCLUDED_PDU_PDU_SPLIT_IMPL_H
+#define INCLUDED_PDU_PDU_SPLIT_IMPL_H
+
+#include <gnuradio/pdu/pdu_split.h>
+
+namespace gr {
+namespace pdu {
+
+class pdu_split_impl : public pdu_split
+{
+private:
+    bool d_pass_empty_data;
```

Comment:
Should it be possible to change that value at runtime? I suggest to make it `const`. as well as the ctor and make signatures. That would convey the intent.